src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -285,12 +285,21 @@ namespace winrt::TerminalApp::implementation
             // launched _fullscreen_, toggle fullscreen mode. This will make sure
             // that the window size is _first_ set up as something sensible, so
             // leaving fullscreen returns to a reasonable size.
+            //
+            // We know at the start, that the root TerminalPage definitely isn't

```

Comment:
But why do we know that? Is it because we reset the focus mode when we hide the quake window?